### PR TITLE
Update files and create a starter Makefile: build_bundler.sh, bundle.c and create a Makefile to start things off

### PR DESCRIPTION
**Pull request by Qwer-TeX**
1. Updated build_bundle.sh to include compiler-found logic - if it cannot find clang, use gcc, if thats not found either, use cc
2. Updated bundle.c to include -h and --help to help show usage and improved file readability.
3. Create a Makefile to get things started. To learn about a thing or two about Makefile (like how I did) click [here](https://makefiletutorial.com/)
4. Update the README.md to be up to date

### CODE REVIEW

File: bundle.c [hunk 1]
```diff
@@ -1,68 +1,98 @@
...
+  int length = strlen(str);
+  char *buf = malloc(length + 2);
+  buf[0] = '_';
+  for (int i = 0; i < length; ++i) {
+    if (isalnum(str[i]))
+      buf[i + 1] = toupper(str[i]);
+    else
+      buf[i + 1] = '_';
+  }
+  buf[length + 1] = '\0';
```
Comment (Qwer-TeX) [inline bundle.c:16]: I think this should be `buf[length + 2] = '\0'`;

Comment (Qwer-TeX) [inline bundle.c:16]: I think this should be `buf[ length + 2 ] = '\0'`
Am I wrong?

File: bundle.c [hunk 2]
```diff
@@ -1,68 +1,98 @@
...
+int main(int argc, char **argv) {
+  if (argc == 2 &&
+      (strcmp(argv[1], "-h") == 0 || strcmp(argv[1], "--help") == 0)) {
+    print_usage();
+    return 0;
+  }
+
+  int files = argc - 1;
+  if (files == 0) {
+    fprintf(stderr, "Error: No files specified.\n");
```
Comment (Qwer-TeX) [inline bundle.c:55]: I added this file guard in so that the user doesn't get confused on thinking to run `./bundle` instead of `./bundle [file1] ...`

**Review by Qwer-TeX — COMMENTED** [review]
That should be it?

**Comment by thisisignitedoreo** [comment]
hi! thank you for pull request, but some things gonna change after merging:
1. my personal nit: i like everything lowercase
2. i dont see a point in build scripts with a mature makefile

**Pull request merged**